File: src/Addins/Eto.Addin.MonoDevelop/Templates/Files/CS/code/preview.form.eto.cs
using System;
using System.Collections.Generic;
using Eto.Forms;
using Eto.Drawing;

namespace ${Namespace}
{	
	partial class ${EscapedIdentifier} : Form
	{	
		void InitializeComponent()
		{
			Title = "My ${EscapedIdentifier} form";

			Content = new StackLayout
			{
				Items =
				{
					new Label { Text = "Some Content" }
				}
			};
		}
	}
}

File: src/Addins/Eto.Addin.VisualStudio.Mac/Templates/Files/FS/code/preview.form.eto.fs
namespace ${Namespace}

open System
open Eto.Forms
open Eto.Drawing

type ${EscapedIdentifier}Base () as this =
    inherit Form ()

    member this.InitializeComponent() =
        base.Title <- "My ${EscapedIdentifier} form"
        base.Content <- new Label (Text = "Some Content")
